modules/ROOT/examples/live-demos/basic-example/index.html

```diff
@@ -1,12 +1,12 @@
 <textarea id="basic-example">
-  {{site.logoForDemosHTML}}
+  {{logofordemoshtml}}
   <h2 style="text-align: center;">Welcome to the TinyMCE editor demo!</h2>
 
   <h2>Got questions or need help?</h2>
 
   <ul>
     <li>Our <a href="https://www.tiny.cloud/docs/">documentation</a> is a great resource for learning how to configure TinyMCE.</li>
```

Comment:
find and replace these https://www.tiny.cloud/docs/
https://www.tiny.cloud/docs/tinymce/6/